client/client.go

```diff
@@ -63,6 +63,25 @@ const (
 	TaskReachabilityClosedWorkflows = internal.TaskReachabilityClosedWorkflows
 )
 
+// UpdateLifeCycleStage indicates the stage of an update request.
+// NOTE: Experimental
+type UpdateLifeCycleStage = internal.UpdateLifeCycleStage
+
+const (
+	// UpdateLifeCycleStageUnspecified indicates the wait stage was not specified
+	// NOTE: Experimental
+	UpdateLifeCycleStageUnspecified = internal.UpdateLifeCycleStageUnspecified
+	// UpdateLifeCycleStageAdmitted indicates the update is admitted
+	// NOTE: Experimental
+	UpdateLifeCycleStageAdmitted = internal.UpdateLifeCycleStageAdmitted
```

Comment:
Wonder if we should leave "admitted" unexposed for now

Reply:
Let me check if the server will ever return admitted from a nonblocking poll

Reply:
Testing with a non blocking poll, the server does expose when the update is admitted

Reply:
Yes we should, I'll do that as part of this PR